JPS_Ontology/ontology/ontocompany/OntoCompany.owl

```diff
@@ -515,384 +651,534 @@
         <rdfs:subClassOf>
             <owl:Restriction>
-                <owl:onProperty rdf:resource="http://www.theworldavatar.com/kg/ontocompany#hasSpecificEnergyConsumption"/>
+                <owl:onProperty rdf:resource="http://www.theworldavatar.com/kg/ontocompany/hasEnergyConsumptionPerUnitFloorArea"/>
                 <owl:qualifiedCardinality rdf:datatype="http://www.w3.org/2001/XMLSchema#nonNegativeInteger">1</owl:qualifiedCardinality>
-                <owl:onClass rdf:resource="http://www.theworldavatar.com/kg/ontocompany#SpecificEnergyConsumption"/>
+                <owl:onClass rdf:resource="http://www.theworldavatar.com/kg/ontocompany/EnergyConsumptionPerUnitFloorArea"/>
+            </owl:Restriction>
+        </rdfs:subClassOf>
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://www.theworldavatar.com/kg/ontocompany/hasSpecificEnergyConsumption"/>
+                <owl:qualifiedCardinality rdf:datatype="http://www.w3.org/2001/XMLSchema#nonNegativeInteger">1</owl:qualifiedCardinality>
+                <owl:onClass rdf:resource="http://www.theworldavatar.com/kg/ontocompany/SpecificEnergyConsumption"/>
             </owl:Restriction>
         </rdfs:subClassOf>
         <rdfs:comment>A manufacturing facility.</rdfs:comment>
-        <rdfs:isDefinedBy>http://www.theworldavatar.com/kg/ontocompany</rdfs:isDefinedBy>
+        <rdfs:isDefinedBy rdf:resource="http://www.theworldavatar.com/kg/ontocompany/"/>
         <rdfs:label>Factory</rdfs:label>
     </owl:Class>
     
 
 
-    <!-- http://www.theworldavatar.com/kg/ontocompany#FlowRate -->
+    <!-- http://www.theworldavatar.com/kg/ontocompany/FactoryProperty -->
+
+
+    <owl:Class rdf:about="http://www.theworldavatar.com/kg/ontocompany/FactoryProperty">
+        <rdfs:subClassOf rdf:resource="http://www.theworldavatar.com/kg/ontocompany/IndustrialFacilityProperty"/>
+        <rdfs:comment>A property of a factory</rdfs:comment>
+        <rdfs:isDefinedBy rdf:resource="http://www.theworldavatar.com/kg/ontocompany/"/>
+        <rdfs:label>factory property</rdfs:label>
+    </owl:Class>
+    
+
+
+    <!-- http://www.theworldavatar.com/kg/ontocompany/FloorArea -->
 
-    <owl:Class rdf:about="http://www.theworldavatar.com/kg/ontocompany#FlowRate"/>
+
+    <owl:Class rdf:about="http://www.theworldavatar.com/kg/ontocompany/FloorArea">
+        <rdfs:subClassOf rdf:resource="http://www.theworldavatar.com/kg/ontocompany/IndustrialFacilityProperty"/>
+        <rdfs:comment>The total floor area of an industrial facility</rdfs:comment>
+        <rdfs:isDefinedBy rdf:resource="http://www.theworldavatar.com/kg/ontocompany/"/>
+        <rdfs:label>floor area</rdfs:label>
+    </owl:Class>
     
 
 
-    <!-- http://www.theworldavatar.com/kg/ontocompany#FoodIndustry -->
+    <!-- http://www.theworldavatar.com/kg/ontocompany/FlowRate -->
+
+
+    <owl:Class rdf:about="http://www.theworldavatar.com/kg/ontocompany/FlowRate"/>
+    
```

Comment:
You mentioned above that the FlowRate class has been removed but it seems to be here